test/integration/serverless/recorder-extension/go.mod

```diff
@@ -36,4 +36,5 @@ replace (
 require (
 	github.com/DataDog/agent-payload/v5 v5.0.19
 	github.com/DataDog/datadog-agent v0.0.0-20211213161047-f82981e22ca1
+	golang.org/x/net v0.1.0 // indirect
```

Comment:
weird that we're adding an indirect dependency here and in that other place. I assume `go mod` did that for you?

Reply:
yes, also this won't reach the binary that we deliver but the static analysis tool was complaining